test/libyul/yulOptimizerTests/redundantAssignEliminator/for_stmnts_after_break_continue.yul

```diff
@@ -0,0 +1,52 @@
+{
+    let x := 1
+    let y := 1
+    for { } calldataload(0) { }
+    {
+        y := 9
+        if callvalue() {
+            x := 2
+            y := 2 // will be removed
+            break
+            x := 7 // after break, we start with fresh state.
+        }
+        if eq(callvalue(), 3) {
+            x := 12
+            y := 12 // will be removed
+            continue
+            x := 17 // after continue, we start with fresh state.
+            y := 9
```

Comment:
Why is `x` removed but not `y`?

Reply:
x is re-assigned in line 20, y is used in line 21 without prior reassignment